code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/identity/Token.java

```diff
@@ -1,8 +1,9 @@
 package io.cattle.platform.iaas.api.auth.identity;
 
 import io.cattle.platform.api.auth.Identity;
+import io.cattle.platform.iaas.api.auth.SecurityConstants;
```

Comment:
The changes in this class were to remove deprecated token creation methods that are no longer needed.